### ISSUE TITLE
Pull in necessary items from Paul's Civic Patterns doc

### ISSUE THREAD

**Issue by longhotsummer** [Code-for-All/civicpatterns.org#1]
https://docs.google.com/document/d/1Pb05LDbmw1UxfYxtsshnQCVXlcubxXH9EkmIe9nLIxo/edit#

**Comment by pmackay** [comment]
On this, any objections to me creating a listing of tools, slightly separate from patterns, as I have at the end of my doc? e.g.
- Hackathon
- Challenge
- Fellow
- etc

**Comment by pudo** [comment]
Sounds great. I also want to import the checklist if that’s ok?

**Comment by longhotsummer** [comment]
:+1: on both.

I also like the list of resources and further reading, we can link to the CodeForAmerica list (they don't have a pretty website like we do)

**Comment by pmackay** [comment]
:+1: +1 for checklist too :)

**Comment by pmackay** [comment]
Added https://github.com/civic-commons/civicpatterns.org/commit/3fdaa8cf0ec3d15a40879612d76a3ebcbfddcfad with initial stubs. 

If any of you Jekyll experts have any suggestions, I'm not too happy about adding the tool.html file as it breaks DRY, but couldnt see a way to write a conditional that could detect the layout of a pattern/tool, cos "layout" was defined to "default" based on the top YAML block I assume.

**Comment by tmtmtmtm** [comment]
@pmackay What would you like to happen instead of needing the tool.html? What is the conditional you'd like to write?

**Comment by pmackay** [comment]
if layout == pattern
Pattern: 
else
Tool: 
end

**Comment by tmtmtmtm** [comment]
I don't think you can access the value of the 'layout' setting in the front matter, as by the time you get to it, it'll have changed to something else. 

You could write the conditional on `if page.category == 'Tools'`, though. 

Alternatively you can avoid duplication by moving the common parts into separate fragments in the _includes direction and {% include fragment-name.html %}

**Comment by pmackay** [comment]
So should this be closed based on https://github.com/Code-for-All/civicpatterns.org/pull/20 and why the change of view? :-O
